frontends/p4/checkNamedArgs.cpp

```diff
@@ -21,7 +21,7 @@ namespace P4 {
 bool CheckNamedArgs::checkArguments(const IR::Vector<IR::Argument> *arguments) {
     bool first = true;
     bool hasName = false;
-    std::map<cstring, const IR::Argument *> found;
+    absl::flat_hash_map<cstring, const IR::Argument *> found;
```

Comment:
Should we use the hasher from `Utils`?

Reply:
It will be used via our `std::hash` specialization. So, nothing special is needed. Though, abseil on top would salt the hash, so the iteration order will be randomized. I do not have preferences which approach is better.

Reply:
After thinking about this a bit more, maybe we can adopt the following rule:
 - For local maps (within the method, etc) we can use `Util::Hash` directly. This way we won't have additional random salt on top of the existing hash, so the hashing latency will be reduced.
 - For members we'd stick with absel's randomized hashes. This way the iteration order will be different on each invocation. As a result, all possible accidentally added iterations could be noticed much earlier.

What do you think?

Reply:
So, I implemented this approach for local maps. Also, found one "funny" case when we had map `cstring -> ID` which I replaces with set.